app/Modules/Recruitment/resources/views/job_advert/index.blade.php

```diff
@@ -0,0 +1,109 @@
+@extends('layouts.main')
+@section('content')
```

Comment:
In cases like this, when the file is copied from another module but not yet adapted, please remove irrelevant data and leave the basic structure only so it's not confusing.